plugin-server/src/worker/ingestion/person-state.ts

```diff
@@ -90,6 +90,12 @@ export class PersonState {
         return this.personContainer
     }
 
+    async updateExceptProperties(): Promise<LazyPersonContainer> {
+        await this.handleIdentifyOrAlias()
+        await this.createPersonIfDistinctIdIsNew()
```

Comment:
This also sets the properties for the person, which can cause things to be slightly odd too potentially

Reply:
Thinking of tests ... 
```
initial persons state: Alice k=v1 ; (can be separate test) Charlie: c = c1 ; Eve: c = e1 
-- these within buffer window:
capture(bob, properties={$set: {k: v3}})  -- creates bob
create_alias(alice, bob)
capture(Eve, properties={$set: {c: e2}})
create_alias(Charlie, Eve)
```
Here are the different options of processing setup: 
1. normal props late, both creation and merges early: k = v1 (because we first process Bob's creation, then merge) ; c = e2
1. normal props late, creation early, merges late (same as everything early / everything late as we can't have a test where we merge, then create as the person as merge would link the ids) k = v1 ; c = c1
1. normal props late, merges early, creation late:  k = v3 (because Bob gets created, but params processed after) ; c = e2

(normal props early, others late isn't an option we'd consider)

Reply:
I think I have a test that works for merges [here](https://github.com/PostHog/posthog/pull/13505/files#diff-e8c057ea332fa67a8f2bc6d811629e71d31b29630bf04fd906509be89f53e398R484)

I can't think of a test that fails for the `createPersonIfDistinctIdIsNew` adding properties out of order case, but I've updated to add a param to not update properties so finders crossed that works.